app/api/__init__.py

```diff
@@ -99,54 +112,53 @@
           '/sessions/<int:id>/relationships/event')
 
 # social_links
-api.route(SocialLinkList, 'social_link_list', '/events/<int:id>/social-links')
+api.route(SocialLinkList, 'social_link_list', '/events/<int:id>/social_links', '/events/<identifier>/social_links')
```

Comment:
Why is this still using underscores, and you just said that every endpoint you modified is working fine